src/undo_log.rs

```diff
@@ -181,6 +184,7 @@ impl<T> Snapshots<T> for VecLog<T> {
     where
         R: Rollback<T>,
     {
+        #[cfg(feature = "log")]
         debug!("rollback_to({})", snapshot.undo_len);
```

Comment:
could you add a custom `debug` macro that does the `cfg`?
like
```rust
macro_rules! debug {
  ($($tt:tt)*) => {
    #[cfg(feature = "log")]
    log::debug!($($tt)*);
  };
}
```
}

Reply:
fixed in [a0fb25d](https://github.com/rust-lang/ena/pull/48/commits/a0fb25d9b2717147c399c5149e9baecb0c3b6e57) and [e32df28](https://github.com/rust-lang/ena/pull/48/commits/e32df28ebd96a1cd3c9d44eda501cf76bde28fb2)